nodeup/pkg/model/context.go

```diff
@@ -484,79 +481,16 @@ func (c *NodeupModelContext) BuildPrivateKeyTask(ctx *fi.ModelBuilderContext, na
 // NodeName returns the name of the local Node, as it will be created in k8s
 func (c *NodeupModelContext) NodeName() (string, error) {
```

Comment:
This function need to return FQDN as it is used for the node cert name.See my previous comment.

Reply:
Okay, i'll return code which return FQDN